src/components/PrivacyTerms.js

```diff
@@ -42,9 +33,8 @@ export const PrivacyTerms = ({ close }) => {
   }, [])
 
   const items = [
-    { title: i18n('settings-usage-reports'), action: onUsageReportConsentSelected },
-    { title: i18n('settings-privacy'), link: 'https://foundation.m.wikimedia.org/wiki/Privacy_policy' },
-    { title: i18n('settings-term'), link: `https://foundation.m.wikimedia.org/wiki/Terms_of_Use/${lang}` }
+    { title: i18n('settings-privacy'), action: goto.privacyPolicy },
+    { title: i18n('settings-term'), action: goto.termsOfUse }
```

Comment:
Listview read the prop `link` to show the link icon next to each item here, the fastest way to do it is adding `link: true` to each item